src/assets/_application.scss

```diff
@@ -0,0 +1,130 @@
+.app-heading {
+  text-align: right;
+  color: $pink;
+  padding: 30px 0;
+}
+.applicationContainer {
+  display: block;
+  max-width: 1024px;
+  margin: auto;
+}
+.app-row {
+  width: 100%;
+  clear: both;
+
+  &:not(:first-child) {
+    padding-top: 32px;
+  }
+  // background: red;
+}
+.col-6 {
+  width: 50%;
+  float: left;
+  margin: 0;
+  box-sizing: border-box;
+
+  &:first-child {
+    padding-right: 8px;
+  }
+  &:last-child {
+    padding-left: 8px;
+  }
+}
+.align-end {
+  align-items: flex-end;
+}
+input {
+  width: 100%;
+  padding: 15px 20px;
+  box-sizing: border-box;
+  outline: none;
+  border-radius: 8px;
+  border: 2px solid #DBE1E8;
+  margin-top: 8px;
+  font-size: 14px;
+
+  &:focus {
+    border: 2px solid $darkBlue;
+  }
+}
+.application-button {
+  width: 100%;
+  padding: 15px 20px;
+  background: #FFFFFF;
+  border: 2px solid #DBE1E8;
+  border-radius: 8px;
+  font-family: $font-body;
+  margin-top: 8px;
+  font-size: 14px;
+  outline: none;
+
+  &:active {
+    border: 2px solid $darkBlue;
+  }
+}
+.Select-control {
+  width: 100%;
+  box-sizing: border-box;
+  outline: none;
+  border-radius: 8px;
+  border: 2px solid #DBE1E8;
+  margin-top: 8px;
+
+  input:active {
+    border: none;
+  }
+}
+.paddingx {
+  padding: 0 16px;
+}
+.paddingr {
```

Comment:
we should have a discussion on slack probably, IMO some variation of [BEM](https://css-tricks.com/bem-101/) usually works well, but open to other suggestions